pkg/proxy/iptables/proxier.go

```diff
@@ -602,6 +581,22 @@ func addHcUpdate(updateMap hcUpdateMap, name types.NamespacedName, isLocal bool)
 	}
 }
 
+func addEndpointToMap(endpointsMap map[string]endpointsInfoSlice, ip string, port *api.EndpointPort, isLocal bool) {
+	// Only add the hostport if the address is valid,
+	// but keep track of port names with invalid addresses
+	// so we can clean them up later
```

Comment:
(following up from comments on previous PR). OK, I see, you're distinguishing `endpointsMap[port.Name]` being implicitly `nil` vs being explicitly set to `nil`. Maybe the comment could clarify that better (the fact that it will affect an iteration over the map later). Also, line width is way too short. (Unless you were aiming for a haiku... "Is address valid? / Remember invalid ports / To clean up later.")